### ISSUE TITLE
Rename uiImage to uiImageAsset

### ISSUE THREAD

**Issue by cody271** [libui-ng/libui-ng#32]

**Comment by rubyFeedback** [comment]
Thank you for your work on patching up libui.

If it is not too much maintenance work on your part, would it be possible to 
mention renaming API changes in the CHANGELOG file as well? The 
one at https://github.com/libui-ng/libui-ng/blob/master/CHANGELOG.md.

I believe this may be helpful for "downstream" language binding maintainers
in different programming languages, simply to see what they may have to
update specifically.

**Comment by cody271** [comment]
> 
> If it is not too much maintenance work on your part, would it be possible to mention renaming API changes in the CHANGELOG file as well?

Absolutely. There are no API changes listed in the changelog yet, only because none have been made yet.
The changelog is using the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) standard format, which requires the `Changed` tag for all breaking API changes. Also any API changes proposed will be clearly marked as such with the label `api breaking change`, as you can see here on this issue #32.

**Comment by cody271** [comment]
I'm trying to delay all work that involves breaking API changes, until we've merged in everything outstanding that does not.
